### PR TITLE
Add and change stuff

### PR DESCRIPTION
**Pull request by Falskim**
- Change assets
- Retiring one row of botes
- Optimization some codes
- Add `auto combat` checker
- Add DEBUG_MODE to tools (to print `find` log)
- Add some bugs (jk)

### CODE REVIEW

File: modules/sortie.py [hunk 1]
```diff
@@ -102,17 +104,38 @@ def clear_mob(self):
                 self.mob_coords = self.look_around('mobs', 2)
             mob_coord = self.filter_mob_coords()
             self.watch_for_distraction(mob_coord)
-            if not self.is_auto_enabled:
-                self.enable_auto()
+            self.start_battle()
+
+    def start_battle(self, is_boss = False):
```
Comment (cytopz) [inline modules/sortie.py]: nice

File: modules/sortie.py [hunk 2]
```diff
@@ -366,27 +371,32 @@ def switch_fleet(self):
...
-        Tools.tap(self.buttons['auto_battle'])
+        print('Enabling auto combat')
+        Tools.tap(Dimension(50, 110))
 
     def evade(self):
+        print('Evading ambush')
         Tools.tap(self.buttons['evade'])
 
     def fail_evade(self):
+        print('Failed to evade')
```
Comment (cytopz) [inline modules/sortie.py]: don't print it here, this method is supposed to check if the evade fails or not. print it inside the `ambush_handler` method instead

File: modules/sortie.py [hunk 3]
```diff
@@ -396,9 +406,13 @@ def filter_retire_ship(self):
     def retire_ship(self):
         print('Retiring ship...')
         Tools.tap(self.buttons['sort'])
-        if not self.has_filtered_retire:
+        if not self.is_retire_filtered:
             self.filter_retire_ship()
-        Tools.tap(self.buttons['tobe_retired_ship'])
+        # Selecting one row botes
+        x = 130
```
Comment (cytopz) [inline modules/sortie.py]: better to use `self.buttons['tobe_retired_ship']`
```suggestion
        ship = self.buttons['tobe_retired_ship']
```

File: modules/sortie.py [hunk 4]
```diff
@@ -396,9 +406,13 @@ def filter_retire_ship(self):
...
+        for i in range(7):
```
Comment (cytopz) [inline modules/sortie.py]:
```suggestion
        for _ in range(7):
           Tools.tap(ship)
           ship.inc_x(130)   # or ship.x += 130
```

File: modules/sortie.py (hunk 2 above)
Comment (Falskim) [reply modules/sortie.py]: Oh right

**Pull request merged**